backend/open_api_spec.yml

```diff
@@ -212,16 +266,14 @@ components:
         - visibility
         - links
         - datasets
-        - created_at
-        - updated_at
```

Comment:
don't we want to substitute in `revised_at` and `published_at`?

Reply:
that required field is only used for json that is in a request, not the response. So we don't need to put revised and published here.